lisa/tools/pkgconfig.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re
+
+from assertpy import assert_that
+from semver import VersionInfo
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+
+coerce_version_info_regex = re.compile(
+    r"v?(?P<major>[0-9]+)\.(?P<minor>[0-9]+)(\.(?P<patch>[0-9]+))?"
+)
+
+
+class Pkgconfig(Tool):
+    @property
+    def command(self) -> str:
+        return "pkg-config"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def install(self) -> bool:
+        assert isinstance(self.node.os, Posix)
+        self.node.os.install_packages("pkg-config")
+        return True
+
+    def _coerce_version_info_string(self, version_string: str) -> VersionInfo:
+        if VersionInfo.isvalid(version_string):
+            return VersionInfo.parse(version_string)
+        # else, attempt to coerce
+        matches = coerce_version_info_regex.search(version_string)
```

Comment:
Does the `utils.parse_version` implement similar functionality?

Reply:
it DOES
moved to using that, this also saves me from writing a regex example 🙂